src/tutorial/meshvis.md

```diff
@@ -30,15 +30,13 @@ and <a href="../fem"><i class="fa fa-book"></i> Finite Element Basics</a> pages
 
 ### <i class="fa fa-check-square-o"></i>&nbsp; Importing meshes from Gmsh and Cubit
 
-The goal of the first step of the  Meshing and Visualization part of the tutorial is to demonstrate the common steps necessary for generating high-quality meshes in Gmsh and Cubit and how to use them in finite element simulations with MFEM. We will start with Gmsh. Users can find more details about Cubit can be found later in this section. 
+First, we will demonstrate the common steps necessary for generating high-quality meshes in [Gmsh](https://gmsh.info/) and [Cubit](https://cubit.sandia.gov/) and how to use them in finite element simulations with MFEM.
 
+Gmsh is an open-source, freely available mesh generation tool with built-in computer-aided design (CAD) functionality and a postprocessor. The input to Gmsh can be a simple text file that provides a description of the geometry of the finite element model. The geometry can be generated using the Gmsh graphical user interface (GUI), simple text editors such as Vi/Vim/Emacs, or using more sophisticated CAD tools such as SolidWorks or Autocad. CAD models in IGES or STEP formats can be imported by the CAD engine of Gmsh, meshed, and prepared as inputs to the MFEM examples. Here, however, we focus on simpler examples showing the process of generating meshes suitable for MFEM and not on the actual geometry. 
 
-Gmsh is an open-source and freely available mesh generation tool with built-in CAD (Computer-aided design) and a postprocessor. The input to Gmsh can be a simple text file that provides a description of the geometry of the finite element model. The geometry can be generated using the Gmsh graphical user interface, simple text editors such as Vi/Vim/Emacs, or using more sophisticated CAD tools, like SolidWorks or Autocad. CAD models in IGES or STEP formats can be imported by the CAD engine of Gmsh, meshed, and prepared as inputs to the MFEM examples. Here, however, we will focus on more simple examples as the focus is on the process of generating meshes suitable for MFEM and not on the actual geometry. 
+Many examples together with documentation on the input syntax can be found at the [Gmsh website](https://gmsh.info/). Users familiar with Gmsh can skip the first steps and download already prepared geometries for meshing.
```

Comment:
Insert a link so users can skip the mesh generation info and go straight to the downloads (line 81)?

Reply:
inserted link to the geometry